pkg/kube/controllers/watchnamespace/controller.go

```diff
@@ -16,11 +16,13 @@ import (
 	"code.cloudfoundry.org/quarks-utils/pkg/config"
 )
 
+// Reconcile represents reconcile.Reconciler
 type Reconcile struct {
 }
 
 var _ reconcile.Reconciler = &Reconcile{}
 
+// Reconcile represents an empty reconciliation
```

Comment:
`// Reconcile doesn't do anything, because there is nothing to reconcile with the watched namespace`